base-template.sh

```diff
@@ -567,52 +689,81 @@ execute_shared_hooks() {
     # split on comma and newline
     IFS="$IFS_COMMA_NEWLINE"
 
-    # Fail if the shared root is not available (if enabled)
-    FAIL_ON_NOT_EXISTING=$(git config --get githooks.failOnNonExistingSharedHooks)
-
     for SHARED_REPO in $SHARED_REPOS_LIST; do
         unset IFS
 
         set_shared_root "$SHARED_REPO"
 
-        if [ ! -f "$SHARED_ROOT/.git/config" ]; then
-            echo "! Failed to execute shared hooks in $SHARED_REPO" >&2
-            echo "  It is not available. To fix, run:" >&2
-            echo "    \$ git hooks shared update" >&2
+        if echo "$EXECUTED_SHARED_HOOKS" | grep "$SHARED_ROOT"; then
+            echo "! Warning: Shared hooks in (type: \`$SHARED_HOOKS_TYPE\`):" >&2
+            echo "  \`$SHARED_REPO\`" >&2
+            echo "  is listed twice and is skipped." >&2
+            ehco "  Check duplicated entries by running:" >&2
```

Comment:
```suggestion
            echo "  Check duplicated entries by running:" >&2
```

Reply:
Do we have a test on this code path? Should it have failed on this typo, or would we just have printed an error halfway through?

Reply:
No I think we dont have a test so far on this code path. I should add this to 117.

Reply:
Done